projects/assets-library/assets/styles/_layout.scss

```diff
@@ -47,28 +47,33 @@
   }
 }
 
+@mixin frame-border {
+  border: 1px solid $color-border;
+}
+
 @mixin card-hover {
   &:hover {
-    @include box-shadow();
+    @include dropdown-box-shadow();
```

Comment:
Can we please move these to its own interaction.scss?

Reply:
https://traceableai.atlassian.net/browse/ENG-5455